aws/Full-PAS-Deployment.json

```diff
@@ -3313,8 +3270,8 @@
             "Components":"ami-0c92618583f701176",
             "CPM":"ami-079a10f6ba5c4eeb4",
             "PVWA":"ami-046e947dcae85aafb",
-            "PSM":"ami-0e6d2cea1cf1d0275",
-            "PSMP":"ami-07f89768f972c452d"
+            "PSM":"ami-0ccf83cef896a80ea",
```

Comment:
Any reason why the AMI Id's changed?

Reply:
PSM AMI id points to PSMP AMI. I have fixed only London, but already requested from Nimrod to check all the others

Reply:
Delete is done as part of "DeletePasswordLambda"

Reply:
"you removed the S3 access from the policy how are the instances going to get the files from the s3 bucket?" - Except Vault and DR no one require access to S3 bucket. Vault and DR have another policy to access license and recpub. Please see: "VaultInstancesS3VaultFilesBucketPolicy"

Reply:
Fixed #2. Consolidated Vault and Component SSM policy